docs/hour-of-code/educators-2021.html

```diff
@@ -0,0 +1,362 @@
+        <!-- TODO: Description Copy -->
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1.0, user-scalable=yes"
+        />
+
+        <link rel="manifest" href="/blb/sim.webmanifest" />
+
+        <meta name="mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta
+            name="apple-mobile-web-app-title"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta
+            name="apple-mobile-web-app-status-bar-style"
+            content="translucent-black"
+        />
+        <meta name="format-detection" content="telephone=no" />
+        <meta
+            name="application-name"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+
+        <meta name="msapplication-TileColor" content="#ff7b22" />
+        <meta
+            name="msapplication-TileImage"
+            content="/static/hour-of-code/card.png"
+        />
+        <meta name="msapplication-config" content="none" />
+        <meta name="theme-color" content="#ff7b22" />
+
+        <meta name="msapplication-TileColor" content="#3DC1AA" />
+        <meta
+            name="msapplication-TileImage"
+            content="/static/hour-of-code/2021/hoc-card.png"
+        />
+        <meta name="msapplication-config" content="none" />
+        <meta name="theme-color" content="#3DC1AA" />
+
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="MSMakeCode" />
+        <meta
+            name="twitter:title"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta name="twitter:description" content="" />
+        <!-- TODO: Description Copy -->
+        <meta
+            name="twitter:image"
+            content="/static/hour-of-code/2021/hoc-card.png"
+        />
+
+        <meta
+            property="og:title"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta
+            property="og:site_name"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta property="og:description" content="" />
+        <!-- TODO: Description Copy -->
+        <meta
+            property="og:image"
+            content="/static/hour-of-code/2021/hoc-card.png"
+        />
+
+        <link
+            rel="apple-touch-icon"
+            href="/static/hour-of-code/2021/favicon.png"
+        />
+        <link
+            rel="icon"
+            type="image/png"
+            href="/static/hour-of-code/2021/favicon.png"
+        />
+        <link
+            rel="shortcut icon"
+            href="/static/hour-of-code/2021/favicon.png"
+        />
+
+        <!-- @include scripthead.html -->
+        <!-- TODO: Font compliance for "Rowdies" Google Font (400 weight) -->
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
^^ @riknoll are we allowed to use google fonts?

Reply:
Yeah, there are two pending requests on Richard in regards to fonts: https://github.com/microsoft/pxt-arcade/issues/3732